velox/functions/lib/aggregates/tests/AggregationTestBase.h

```diff
@@ -81,6 +81,15 @@ class AggregationTestBase : public exec::test::OperatorTestBase {
           exec::test::AssertQueryBuilder& builder)> assertResults,
       const std::unordered_map<std::string, std::string>& config = {});
 
+  void testSingleAggregation(
```

Comment:
What's the point of adding this method? It seems the calling code would be more readable without it.

Reply:
Multi input distinct only works with SingleAggregation.

Thus, in the new test logic added to velox/functions/prestosql/aggregates/tests/CovarianceAggregationTest.cpp, we need a way to call only the "test single aggregation" logic of AggregationTestBase class.

Reply:
Sure, but this means that all you are doing is running a single query plan. You can do this directly, no?